virttest/storage.py

```diff
@@ -233,6 +234,8 @@ def get_image_filename(params, root_dir, basename=False):
 
             return storage_ssh.get_image_filename(server, ssh_image_path,
                                                   user, port, host_key_check)
+        if storage_type == 'iscsi' and image_raw_device == 'yes':
+            return image_name
```

Comment:
this depends on the env_process, qemu_storage/storage can be used without starting vm, so we cannot assume that params has been updated in env_process here.

if storage_type == 'iscsi':
     return iscsi.get_image_filename_iscsi(params)

In iscsi.get_image_filename_iscsi, we log into iscsi server and scan the devices(get_devcies) then we get the image name (e.g. /dev/sdb) with lun id.

The same issue may happen on nfs, please make a note for future fix.

Reply:
Have updated, please help to check again.